vm/starnix/starnix.go

```diff
@@ -277,20 +311,35 @@ func (inst *instance) Forward(port int) (string, error) {
 }
 
 func (inst *instance) Copy(hostSrc string) (string, error) {
+	startTime := time.Now()
 	base := filepath.Base(hostSrc)
-	vmDst := filepath.Join(targetDir, base)
 	if base == "syz-fuzzer" || base == "syz-execprog" {
 		return hostSrc, nil // we will run these on host.
 	}
+	vmDst := filepath.Join(targetDir, base)
 
-	err := inst.runCommand(
-		"adb",
-		"-s",
-		fmt.Sprintf("127.0.0.1:%d", inst.port),
-		"push",
-		hostSrc,
-		vmDst)
-	return vmDst, err
+	for {
+		vmimpl.SleepInterruptible(3 * time.Second)
```

Comment:
Just checking: do we want this sleep even before the first attempt?

Reply:
Good point, it used to be helpful, but now that we have retries we might as well try once right away.